scipy/integrate/quadpack.py

```diff
@@ -560,6 +560,21 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
     romb : integrator for sampled data
     scipy.special : for coefficients and roots of orthogonal polynomials
 
+    Examples
+    --------
+
+    Calculate :math: double integration exp(-x*y)/y**5
+    Limits from 0 to inf
+
+    >>> from scipy import integrate
+    >>> from numpy as np
```

Comment:
This line is implicit in all docstring examples.

Reply:
I will remove these lines

Reply:
`integrate` import should be kept, only `import numpy as np` is implicit.

Reply:
Done